src/cpp/client/secure_credentials.cc

```diff
@@ -61,6 +62,7 @@ std::shared_ptr<Credentials> WrapCredentials(grpc_credentials* creds) {
 }  // namespace
 
 std::shared_ptr<Credentials> GoogleDefaultCredentials() {
+  GrpcLibrary init;  // To call grpc_init().
```

Comment:
From https://github.com/grpc/grpc/blob/master/include/grpc/grpc.h#L378, it seems we should do it in the C++ layer. Maybe even add it to all the XXCredentials() functions just to be safe?